packages/loot-core/src/types/models/rule.d.ts

```diff
@@ -11,7 +11,19 @@ export interface RuleEntity {
 
 interface RuleConditionEntity {
   field: unknown;
-  op: unknown;
+  op:
+    | is
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I think these should be strings. Otherwise it's just `op: unknown | unknown | unknown`

```suggestion
    | 'is'
```